src/Network/Network.Test/Network.Test.csproj

```diff
@@ -14,12 +14,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="601.1939.2547-preview" />
```

Comment:
It looks like this is not standard released SDK, and this cannot be dependency for azure-powershell release

Reply:
I got this SDK from Anton. He generated this out of the SDK pipeline.

Reply:
This is temp SDK which should be use for development until we get a proper SDK. I shared signed SDK with Tejas to use in this PR

Reply:
Updated the PR with the signed SDK. Thanks Anton for the SDK.